Code/Core/Process/Process.h

```diff
@@ -7,6 +7,10 @@
 #include "Core/Env/Types.h"
 #include "Core/Containers/AutoPtr.h"
 
+#if defined( __WINDOWS__ )
+    #include <windows.h>
+#endif
```

Comment:
FASTBuild's core libraries go out of their way to keep windows.h our of the headers to avoid compile time impact.